core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/TestCASLockFactory.java

```diff
@@ -135,6 +140,8 @@ public void testGlobalLockTakenThrowsException() throws LockException
         try (DistributedLock lock = myLockFactory.tryLock(null, "lock", 1, new HashMap<String, String>()))
         {
         }
+
+        assertPriortiesInList("lock", 1);
```

Comment:
Will not be reached. Consider this:
http://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#exception-assertion

Or just a "finally"

Reply:
Have a patch on the way to use #assertThatExceptionOfType(...).isThrownBy(...)

Will use finally for now